### ISSUE TITLE
Not working on Anki Version 2.1.24 (dev)

### ISSUE THREAD

**Issue by MelihDarcanxyz** [ssricardo/anki-plugins#60]
I've installed Fill the Blanks add-on and replaced {{cloze:Text}} with {{type:cloze:Text}} But it keeps saying "The front of this card is blank."
![Screenshot_2020-03-22_21-53-14](https://user-images.githubusercontent.com/57872471/77257829-c59ce500-6c87-11ea-8ade-270f08625904.png)
This is how it is normally.
![Screenshot_2020-03-22_21-52-41](https://user-images.githubusercontent.com/57872471/77257842-d77e8800-6c87-11ea-9008-90fdceea1f92.png)
This is how it looks after add-on installed and "type" added. How can I fix this?
Thank you for reading and making such a great add-on.
Edit#1
Tried on Anki Version 2.1.24 (9dda5cf6) too. The result is same either.
Edit#2
It was working on Anki Version 2.1.15.
Edit#3
I've noticed that if I put "type" in second card it looks like this. But still not working with first card.
![Screenshot_2020-03-23_02-08-35](https://user-images.githubusercontent.com/57872471/77264478-77014200-6cab-11ea-8db5-55c124440bad.png)

**Comment by exdeval** [comment]
Tried on Anki Version 2.1.24 (9dda5cf6) too. The result is same either.

**Comment by ssricardo** [comment]
Version .24 already?! I don't even have that version yet, and it's a little hard to go along with all versions... 

Don't need it already or can you use current version .22 for now?

I'll try to check it, but I don't know yet when, ok?!

**Comment by MelihDarcanxyz** [comment]
Thank you for your reply. I've tested it with Version 2.1.20 (47a1bf8b) an hour ago and it's working pretty nice. So we can say the problem is solved. I've noticed few things (small things but I just wanted to give a feedback).
![Screenshot_2020-03-24_01-52-19](https://user-images.githubusercontent.com/57872471/77370599-49330080-6d72-11ea-8d7b-cb5be559d267.png)
![Screenshot_2020-03-24_01-52-40](https://user-images.githubusercontent.com/57872471/77370612-5223d200-6d72-11ea-990f-087316b2ab81.png)
Is there a way to balance how big the input field with the actual answer? Anyway, the issue is kinda solved. It's just very recent version of Anki. Thank you again!
